app/page/index.js

```diff
@@ -111,20 +271,21 @@ const startAppService = (token) => {
 			console.log('service started complete_func:', JSON.stringify(info));
 			hmUI.showToast({
 				text: info.result
-					? 'Service started.'
-					: 'Service failed to start; check if logged in via settings.',
+					? 'Service started' + token
+					: 'Service failed to start',
 			});
 		},
 	});
 };
 
-const checkPermissions = () => {
-	const [permissionResult] = queryPermission({
-		permissions,
-	});
+// Asynchronous permission check
+const checkPermissions = async () => {
```

Comment:
why does this need to be async?

Reply:
i mainly did it because of the query and request permissions stuff, but looking back the two don't really need it since we dont need to call await or anything. i will change it to get rid of the async